playground/worker/__tests__/basic-functionality.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { getTextResponse, isBuild, serverLogs } from '~utils';
+import { expect, test } from 'vitest';
+
+test.runIf(!isBuild)('basic hello-world functionality', async () => {
+	expect(await getTextResponse()).toMatchInlineSnapshot(`"Hello World!"`);
+});
+
+test.runIf(!isBuild)('basic dev logging', async () => {
+	expect(serverLogs.info.join()).toContain('__console log__');
+	expect(serverLogs.errors.join()).toContain('__console error__');
+	expect(serverLogs.errors.join()).toContain('__console warn__');
```

Comment:
Note: For complete transparency I want to point out a flaw of this testing strategy here, counting the number of logs like these is tricky since always `goTo`ing into the page, so here if you were to see the `serverLogs` you'd find two `__console log__` messages, I am not sure if there is a clean/nice way to avoid such issue, but hopefully it's not a huge limitation here (in this test I simply did `join` + `toContain` and that works well enough)